### DISCUSSION TITLE
"http://localhost:5500"/ping

### DISCUSSION THREAD

**Discussion by FThelin** [DevSkillsHQ/discusssions#60] [Help] [answered]
My tests passes locally, but when I push, the job "Run tests with waiting on API" fails, saying:
Unable to locate executable file: "http://localhost:5500"/ping. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.

**Comment by ludvig** [comment, marked as answer]
Hi, it seems to be a potential issue with how the github workflow parses the api url from the provided `cypress.config.js` file. In this file the strings are expected to be formatted with single quotations `'`, could you please make sure you are using single quotation strings for this config file?

**Reply by FThelin** [reply]
omg I actually tried that, but prettier changed it back to double quotes. I had to disable it. Now it works fine. Thank you!
